src/components/NetworkMap.vue

```diff
@@ -3,71 +3,137 @@
     <h3>Network Map</h3>
     <div class="map-container">
       <div class="wrapper">
-        <img ref="mapImage" src="/assets/mercator-map.png" alt="">
-        <canvas ref="mapCanvas"></canvas>
+        <img ref="mapImage" src="/assets/world-robinson.svg" alt="">
+        <svg ref="mapMarkers" id="mapMarkers"></svg>
       </div>
     </div>
   </div>
 </template>
 
 <script>
+// The network map uses a Robinson projection map image with slight cropping at the top and bottom of the map - https://simplemaps.com/resources/svg-world
+// co-ordinates are calculated with the following formulas:
+// x = 0.8474 RX (lng - lng0)
+// y = 1.3523 RY
+// - R is radius of globe at scale of the map, where equator length is 0.8487 of map width
+// - X and Y are coefficient values determined by Robinson at known latitudes, with intermediate values calculated by interpolation
+// (I have used linear interpolation for simplicity, at slight cost of accuracy)
+// the formula and coefficients can be found here: https://en.wikipedia.org/wiki/Robinson_projection#Formulation
+
+// known X and Y coefficients for latitudes at 5 degree intervals
+const latArray = [0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90]
+const XValues = [1, 0.9986, 0.9954, 0.9900, 0.9822, 0.9730, 0.9600, 0.9427, 0.9216, 0.8962, 0.8679, 0.8350, 0.7986, 0.7597, 0.7186, 0.6732, 0.6213, 0.5722, 0.5322]
+const YValues = [0, 0.0620, 0.1240, 0.1860, 0.2480, 0.3100, 0.3720, 0.4340, 0.4958, 0.5571, 0.6176, 0.6769, 0.7346, 0.7903, 0.8435, 0.8936, 0.9394, 0.9761, 1]
```

Comment:
I think this might be better expressed as one array of tuples, i.e.

```js
// for want of a better name
const lxy = [
  [0, 1, 0],
  [5, 0.9986, 0.0620],
  [10, 0.9954, 0.1240],
  // ...etc...
]
```

Alternatively, align the values in the three arrays with additional spaces so that they are readable in tabulated form.

Reply:
Also, don't use capital letters at the start of constant/variable names. The JS language spec doesn't have a problem with this but the convention in our code is CapitalNames for abstracts (e.g. class names, types) and camelCase for variables. The exception to this rule would be an ALL_CAPS_CONSTANT but as the data is object-type I'd steer clear in this case.

XValues and YValues could be more descriptively named, also (if not condensed into a single array).

Reply:
Just to feed back on this and the tuples (or truples), these arrays are typically how these algorithms have been implemented, and the values are all quite arbitrary, not necessarily meant for human consumption, and won't be modified either, so I don't think it's of import to change how they're represented here. In terms of long lines, they could always be broken down onto multiple lines, for example:

```
const latArray = [
  0, 5, 10, 15, 20, 25, 30, 35, 40, 45,
  50, 55, 60, 65, 70, 75, 80, 85, 90
]
```

Reply:
By tabulated form I mean aligning values to resemble a spreadsheet:

```
const latArray = [0, 5,      10,      15, ...]
const XValues =  [1, 0.9986,  0.9954,  0.9900, ...]
const YValues =  [0, 0.0620,  0.1240,  0.1860, ...]
```

This is possibly very petty feedback and I'm fine with it remaining as-is if it's 2 against 1.

Reply:
@annybs 
Regarding the capital at start of `XValues` and `YValues`, this is only because `x` and `X` are two different things in the formula, so I needed to differentiate between them. `x` is the co-ordinate, as you'd expect, where as `X` is a sort of coefficient that is used to get from longtitude to `x`. I've seen them named A and B elsewhere which I guess was to help avoid confusion. 

With this in mind, are you ok me leaving as capitals?